src/AerospikeCache.php

```diff
@@ -0,0 +1,127 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+use Symfony\Component\Cache\Exception\CacheException;
+
+class AerospikeCache extends AbstractAdapter
+{
+    private const WRAPPER_NAME = 'data';
+
+    /** @var \Aerospike */
+    private $aerospike;
+
+    /** @var string */
+    private $namespace;
+
+    /** @var string */
+    private $set;
+
+    public function __construct(
+        \Aerospike $aerospike,
+        string $namespace,
+        string $set = 'cache',
+        string $cacheNamespace = '',
+        int $defaultLifetime = 0
+    ) {
+        $this->aerospike = $aerospike;
+        $this->namespace = $namespace;
+        $this->set = $set;
+        parent::__construct($cacheNamespace, $defaultLifetime);
+    }
+
+    protected function doFetch(array $ids): array
+    {
+        $result = [];
+
+        $keys = $this->initializeKeysForAerospike($ids);
+
+        $this->aerospike->getMany($keys, $records);
+
+        foreach ($records as $record) {
+            if ($record['metadata'] !== null) {
+                $result[$record['key']['key']] = isset($record['bins'][self::WRAPPER_NAME]) ?
+                    $record['bins'][self::WRAPPER_NAME] : null;
+            }
+        }
+
+        return $result;
+    }
+
+    private function initializeKeysForAerospike(array $ids): array
+    {
+        return array_reduce(
```

Comment:
Since you create same size array on the output, it should just be `array_map`..

Reply:
changed